spec/spec_helper.rb

```diff
@@ -6,6 +6,7 @@
 Pry.config.color = false
 
 require "cc/logger"
+require "cc/engine/duplication"
```

Comment:
Not clear to me why this was needed here, but there are extant `requires` for this in other specs that can be removed if we're adding it here.

Reply:
Yeah, I ran into a issue when using `RSPEC_ARGS` without this higher level require. Can remove the spec specific ones.

Reply:
Updated.